chromium/pages/cancel/style.css

```diff
@@ -1,20 +1,34 @@
 body{
   margin-top: 6em;
-  left: 50%;
   position: relative;
-  overflow: hidden;
+  text-align: center;
 }
 
-h1 img{
-  position: relative;
-  left: -289px;
+.content{
+  width: 600px;
+  margin: auto;
+  text-align: left;
+}
+
+h1{
+  text-align: center;
 }
 
 p{
-  width: 600px;
-  position: relative;
-  left: -300px;
   font-size: 12pt;
   font-family: sans-serif;
   line-height: 150%;
 }
+
+form, form button{
+  font-size: 12pt;
+}
+
+form button{
+  padding: 8px 16px;
+}
+
+#open-url-button{
+  background: red;
```

Comment:
```css
button {
	cursor: pointer;
	display: inline-block;
	margin: 0;
	padding: 1em 1em;
	background: repeating-linear-gradient( 45deg, red, red 7.5px, salmon 7.5px, salmon 15px );
	color: #fff;
	font-weight: bold;
	text-align: center;
	vertical-align: middle;
	-moz-user-select: none;
  user-select: none;
	border: 0;
	border-radius: 5px;
}
```

Reply:
Thanks for providing the CSS. The screenshot has been updated.